lib/cache.js

```diff
@@ -71,13 +71,13 @@ function getOpportunities() {
       orgImages: matterResult.data.orgImages,
       orgName: matterResult.data.orgName,
       lat: matterResult.data.lat ? matterResult.data.lat : "-1000",
-      long: matterResult.data.lon ? matterResult.data.long : "-1000",
+      long: matterResult.data.long ? matterResult.data.long : "-1000",
       linkTo: matterResult.data.linkTo ? matterResult.data.linkTo : "",
       city: matterResult.data.linkTo ? matterResult.data.city : "Anywhere",
       address: matterResult.data.address,
       tags: matterResult.data.tags,
-      published: matterResult.data.published ? matterResult.data.published : true,
-      archived: matterResult.data.archived ? matterResult.data.archived : false,
+      published: matterResult.data.published === "Not" ? false : true,
+      archived: matterResult.data.archived === "Not" ? false : true,
```

Comment:
Archived actually works differently :P default is true so instead of Not make it like Archived or something... Idk if not is the best string either so feel free to change that to like unpublished.